### PR TITLE
Add initial Bring up Milestones markdown.

### PR DESCRIPTION
**Pull request by russellhadley**
This adds a place for us to keep milestones that will take a PR.
These milestones will be linked to out of the wiki for reference. For
additions open a github issue to start a conversation then PR an edit
that has the new milestone.  When milestones are acheived strike through the txt.

### CODE REVIEW

**Comment by russellhadley** [comment]
@cmckinsey Near term milestones for LLILC.

**Comment by russellhadley** [comment]
@libengu Could this be a missed exclusion? There's no way my doc change triggered this.

File: Documentation/llilc-milestones.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# LLILC Bring up Milestones
+
+To meet the overarching goal of fully functional JIT and AOT code 
+generators we've broken out some intermediate milestones that we 
+think are good steps along the way.  Each of these milestones represent 
+a new level of functionality or robustness on the way to a production 
+quality tool.
+
+### Milestones
+* **"Hello World"** - Using LLILC, JIT the all the methods for runtime start
```
Comment (erozenfeld) [inline Documentation/llilc-milestones.md]: Typos: the all the --> all the, start up --> start-up

File: Documentation/llilc-milestones.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+   exits cleanly.
+* **corefx tests on Windows** - Run corefx unit tests as an integration test. This
+  tests a broad set of framework functionality. Success is a clean
+  test run using run-test from the corefx repo.
+* **corefx tests on Linux** - Same as above just on Linux.
+* **JIT SelfHost on Linux** - Pass ~6k Jit SelfHost tests on Linux.
+  These tests are currently being added to coreclr. Success will
+  be running clean for the whole set.
+* **JIT SelfHost Stress on Linux** - Pass ~6k JIT SelfHost tests under GCStress
+  this will validate GC implementation.
```
Comment (erozenfeld) [inline Documentation/llilc-milestones.md]: Two sentences in one.

**Comment by JosephTremoulet** [comment]
Do you want to include mini roslyn, roslyn on windows, anything from #13, or anything about GC?

**Comment by JosephTremoulet** [comment]
How are we going to tag issues?  We've been using "needed for xxx" labels for this sort of thing, and using milestones for sprints.  Are we going to start using milestones for this sort of thing?  If we're going to keep using labels, it would be worth spelling that out here since otherwise the terminology is confusing.

**Comment by russellhadley** [comment]
@JosephTremoulet I'm going to need to follow up with a wiki doc on the tagging/issues/workflow.  This is just the milestones in a form where we can PR.  I was going to do the framing document in the wiki since that seems a think we don't need PRs.  Actually I need to put in the check box think you did.  (from the link you put in)

**Comment by russellhadley** [comment]
I'm inclined to omit the mini-roslyn since we're going to have the full one soon and I don't think there's a big gap.  Also I'm more inclined to focus on the one that is the big milestone for management.  Obviously I expect the Windows one to work before the Linux one but I chose to focus on Linux.

**Comment by libengu** [comment]
retest this please

**Comment by libengu** [comment]
@russellhadley I think it is because this PR fell into the gap between https://github.com/dotnet/coreclr/pull/766 and https://github.com/dotnet/llilc/pull/481. I just sent this PR for retest.

**Comment by russellhadley** [comment]
I've addressed review feed back.  If there's nothing more I'm going to merge.

**Comment by libengu** [comment]
@russellhadley  You don't need to wait for CI result to merge.

**Comment by russellhadley** [comment]
@libengu Yep.  I'm just giving a moment in case anybody want's to add an additional comment.

**Pull request merged**